### PR TITLE
msp: update ops with deployment info

### PR DESCRIPTION
**Pull request by jac**
Updates MSP ops pages to include deployment info and links to delivery pipelines if applicable

Deployed in this handbook pr: https://github.com/sourcegraph/handbook/pull/8845
## Test plan

docs changes

### CODE REVIEW

File: dev/managedservicesplatform/operationdocs/operationdocs.go [hunk 1]
```diff
@@ -83,29 +83,47 @@ This service is operated on the %s.`,
...
+		for _, environment := range s.Environments {
+			if environment.ID == finalStageEnv {
+				finalStageProject = environment.ProjectID
+				break
+			}
+		}
+		serviceDetails = append(serviceDetails,
+			[]string{"Rollout Pipeline", markdown.Linkf(fmt.Sprintf("`%s-%s-rollout`", s.Service.ID, region),
+				"https://console.cloud.google.com/deploy/delivery-pipelines/%[1]s/%[2]s-%[1]s-rollout?project=%[3]s", region, s.Service.ID, finalStageProject)})
+	}
```
Comment (bobheadxi) [inline dev/managedservicesplatform/operationdocs/operationdocs.go:122]: I wonder if we should have a dedicated section for rollout pipelines as well? We can list the environments and the order, and also say that 2nd+ stages must be manually promoted by doing X, Y, Z etc

File: dev/managedservicesplatform/operationdocs/operationdocs.go [hunk 2]
```diff
@@ -83,29 +83,47 @@ This service is operated on the %s.`,
...
+	region := "us-central1"
+	if s.Rollout != nil {
+		finalStageEnv := s.Rollout.Stages[len(s.Rollout.Stages)-1].EnvironmentID
+		var finalStageProject string
+		for _, environment := range s.Environments {
+			if environment.ID == finalStageEnv {
+				finalStageProject = environment.ProjectID
+				break
+			}
+		}
```
Comment (bobheadxi) [inline dev/managedservicesplatform/operationdocs/operationdocs.go:118]: FYI you can do `s.GetEnvironment(finalStageEnv)`

File: dev/managedservicesplatform/operationdocs/operationdocs.go [hunk 3]
```diff
@@ -83,29 +83,47 @@ This service is operated on the %s.`,
...
+		serviceDetails = append(serviceDetails,
+			[]string{"Rollout Pipeline", markdown.Linkf(fmt.Sprintf("`%s-%s-rollout`", s.Service.ID, region),
```
Comment (bobheadxi) [inline dev/managedservicesplatform/operationdocs/operationdocs.go:120]: To align with the others, and I think we generally use sentence casing for titles:

```suggestion
			[]string{"Rollout pipeline", markdown.Linkf(fmt.Sprintf("`%s-%s-rollout`", s.Service.ID, region),
```

File: dev/managedservicesplatform/operationdocs/operationdocs.go (hunk 1 above)
Comment (jac) [reply dev/managedservicesplatform/operationdocs/operationdocs.go]: What do you think of? https://github.com/sourcegraph/sourcegraph/pull/61610/commits/6e02cd668a8b2beaf41359e96339bde4d95e478f

**Review by unknwon — APPROVED** [review]

File: dev/managedservicesplatform/operationdocs/operationdocs.go [hunk 4]
```diff
@@ -122,6 +124,32 @@ This service is operated on the %s.`,
 		md.Paragraphf(strings.Join(lines, "\n"))
 	}
 
+	if s.Rollout != nil {
+		md.Headingf(2, "Rollout")
```
Comment (bobheadxi) [inline dev/managedservicesplatform/operationdocs/operationdocs.go:128]:
```suggestion
		md.Headingf(2, "Rollouts")
```

File: dev/managedservicesplatform/operationdocs/operationdocs.go (hunk 1 above)
Comment (bobheadxi) [reply dev/managedservicesplatform/operationdocs/operationdocs.go]: LGTM, made a minor suggestion to the header - thank you!

**Review by bobheadxi — APPROVED** [review]
Nice, thank you!

**Pull request merged**